src/EditorFeatures/Core.Wpf/InlineHints/InlineHintsTagger.cs

```diff
@@ -131,21 +129,29 @@ public IEnumerable<ITagSpan<IntraTextAdornmentTag>> GetTags(NormalizedSnapshotSp
                     var dataTagSpans = tag.Span.GetSpans(snapshot);
                     if (dataTagSpans.Count == 1)
                     {
-                        var dataTagSpan = dataTagSpans[0];
-                        var parameterHintUITag = InlineHintsTag.Create(
-                            tag.Tag.Hint, Format, _textView, dataTagSpan, _taggerProvider, _formatMap, classify);
-
-                        _cache.Add(new TagSpan<IntraTextAdornmentTag>(dataTagSpan, parameterHintUITag));
+                        _cache.Add((tag, null));
                     }
                 }
             }
 
+            var document = snapshot.GetOpenDocumentInCurrentContextWithChanges();
+            var classify = document?.Project.Solution.Workspace.Options.GetOption(InlineHintsOptions.ColorHints, document?.Project.Language) ?? false;
```

Comment:
```suggestion
            var classify = document is null
                ? false
                : document.Project.Solution.Workspace.Options.GetOption(InlineHintsOptions.ColorHints, document.Project.Language);
```

Reply:
I prefer to keep the original form here so it's clear this is the same code just moved down.